src/components/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInstagram, faFacebookSquare, faWhatsapp } from '@fortawesome/free-brands-svg-icons';
+
+import '../assets/styles/components/Footer.css';
+
+class Footer extends React.Component{
```

Comment:
I would suggest using functional components here given that there is not too much logic.

Reply:
Done, the Footer is now a functional component